sw/device/lib/dif/dif_entropy_src.h

```diff
@@ -26,6 +26,11 @@ enum {
    */
   // TODO: Synchronize value with hardware.
   kDifEntropySrcFifoMaxCapacity = 64,
+
+  /**
+   * Default firmware override FIFO threshold.
+   */
+  kDifEntropyFifoIntDefaultThreshold = 32,
```

Comment:
nit: what does "Int" stand for? Interrupt? (based on the comment for `kDifEntropySrcInvalidFwOverrideBufferThreshold`)